minifi/pom.xml

```diff
@@ -449,10 +449,22 @@ limitations under the License.
                 <version>1.17.0-SNAPSHOT</version>
             </dependency>
 
+            <!-- C2 modules -->
+            <dependency>
+                <groupId>org.apache.nifi</groupId>
+                <artifactId>c2-client-api</artifactId>
+                <version>1.17.0-SNAPSHOT</version>
+            </dependency>
+            <dependency>
+                <groupId>org.apache.nifi</groupId>
+                <artifactId>c2-client-base</artifactId>
+                <version>1.17.0-SNAPSHOT</version>
+            </dependency>
+
             <dependency>
                 <groupId>com.squareup.okhttp3</groupId>
                 <artifactId>okhttp</artifactId>
-                <version>3.12.3</version>
+                <version>${okhttp.version}</version>
```

Comment:
This version can be removed.

Reply:
It looks like this line still needs to be removed.

Reply:
My bad, I removed the version but it caused build issues and added back and forgat to comment back. So I removed the whole dependency as is comes from parent, without the version but keeping the dependency here overrides the one coming from nifi and looks for the missing version.